tools/c7n_azure/tests/test_output.py

```diff
@@ -43,7 +43,7 @@ def get_azure_output(self, custom_pyformat=None):
             ExecutionContext(
                 None,
                 Bag(name="xyz", provider_name='azure'),
-                Config.empty(output_dir=output_dir)
+                Config.empty(output_dir=output_dir, options={})
```

Comment:
could you clarify why the empty options was needed?

Reply:
it caused the test to fail without it since the context class would not have an options attribute. 
The test fails because it checks for `self.ctx.options.get("log_type")`